Covid-19Radar/Covid19Radar
# Change button state by a value of diagnosis UID

## Purpose

Change button state by a value of diagnosis uid. The user will be able to know what to do next depending on the input status of diagnosis UID.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: UI Improvement
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
1. Open "陽性情報の登録" > "同意して陽性登録する"
2. Confirm that "登録する" is disabled in first.
3. Input arbitrary 1~7 digit numbers.
4. Confirm that "登録する" remains disabled.
5. Input arbitrary 8 digit numbers.
6. Confirm that "登録する" becomes enabled.
```

## What to Check
Verify that the following are valid
* When a digit of diagnosis UID are 0~7, "登録する" has been disabled.
* When a digit of diagnosis UID is 8, "登録する" has been enabled.

## Other Information
